app/views/sessions/new.html.erb

```diff
@@ -11,9 +11,14 @@
         <%= govuk_button_to t(".sign_in_with_dsi"), omniauth_sign_in_path("dfe") %>
       <% end %>
 
-      <p class="govuk-body">
-        <%= sanitize t(".disclaimer", service_name:, support_email: mail_to(t("#{current_service}.support_email"))) %>
-      </p>
+      <% if current_service == :placements %>
+        <h2 class="govuk-heading-m">How to get an account</h2>
+        <p class="govuk-body">This is a pilot service for schools and teacher training providers in Leeds.</p>
+        <p class="govuk-body">Ask your colleague in your organisation to add you if you do not have an account.</p>
```

Comment:
```suggestion
        <p class="govuk-body">Ask a colleague in your organisation to add you if you do not have an account.</p>
```

Reply:
@carlosmartinez Thanks for catching that!